backend/notifications/backends/local.py

```diff
@@ -32,3 +32,30 @@ def send_email(
         print("=== End Email sending ===")
 
         return f"messageid-{uuid4()}"
+
+    def send_raw_email(
```

Comment:
could we migrate to django send_mail and remove our custom wrapper now that we don't have microservices anymore?

todo: check django_ses gives us back the message-id